pkgs/applications/office/rmapi/default.nix

```diff
@@ -0,0 +1,34 @@
+{ callPackage, fetchFromGitHub, buildGoModule, buildEnv, stdenv}:
+
+let
+  pname = "rmapi";
+  version = "0.0.7";
+
+  repo = fetchFromGitHub {
+    owner = "juruen";
+    rev = "tags/v${version}";
+
+    repo = pname;
+    sha256 = "0m51gmrsdwasrlamg70186f22lb21pw79vkrqr13iqg2cszdlrva";
+  };
+in
+# The idea of the following was to keep the derivation.nix and deps.nix
+# in juren/rmapi. Sadly, this does not work becuase hydra does not support
+# import-from-derivation. For this reason, we have to manually copy the
+# two files into nixpkgs.
+# Reference:
+# * https://discourse.nixos.org/t/is-importing-a-derivation-from-another-repository-bad-practice/5016/4?u=enteee
+#  callPackage "${repo}/derivation.nix" {
+
+  callPackage ./derivation.nix {
+    buildGoModule = super: buildGoModule (super // {
```

Comment:
I'm not sure I understand what's "import from derivation", but I think the meaning is that using simply `buildGoModule` as is doesn't work?

Reply:
No that's not what IFS means. I updated the pull request with a link to the wiki which explains IFS derivations. I am nor 100% sure if I should leave this comment or if I should get rid of it entirely.